public_html/wp-content/plugins/wcpt/wcpt-event/class-event-loader.php

```diff
@@ -17,6 +17,7 @@ public function __construct() {
 		add_action( 'plugins_loaded', array( $this, 'includes' ) );
 		add_action( 'init', array( $this, 'register_post_types' ) );
 		add_action( 'init', array( $this, 'register_post_statuses' ) );
+		add_action( 'change_locale', array( $this, 'register_post_statuses' ) ); // Re-register them when the locale changes, to have localised statuses.
```

Comment:
```suggestion
		add_action( 'init', array( $this, 'register_post_types' ) );
		add_action( 'init', array( $this, 'register_post_statuses' ) );
		// Re-register them when the locale changes, to have localised statuses.
		add_action( 'change_locale', array( $this, 'register_post_types' ) );
		add_action( 'change_locale', array( $this, 'register_post_statuses' ) );
```

The post types can probably be reloaded too, to ensure they're properly translated, but I don't think they're used anywhere in logs so are likely less problematic.